exasol/exaslpm/pkg_mgmt/install_apt.py

```diff
@@ -0,0 +1,17 @@
+import subprocess
+
+from exasol.exaslpm.model.package_file_config import AptPackages
+
+
+def install_via_apt(apt_packages: AptPackages):
+    if apt_packages is not None:
+        for package in apt_packages.packages:
+            command_01 = "apt-get -y update"
+            command_02 = "apt-get -y clean"
+            command_03 = "apt-get -y autoremove"
+            command_04 = f"apt-get install -V -y --dry-run --no-install-recommends {package.name}"
```

Comment:
We should add all packages as parameters to a single apt-get install command.
Is the --dry-run for testing?